Scripts/Services/Expansions/Time Of Legends/Mobiles/Dimetrosaur.cs

```diff
@@ -115,7 +115,7 @@ public Dimetrosaur(Serial serial)
         public override void Serialize(GenericWriter writer)
         {
             base.Serialize(writer);
-            writer.Write(1);
+            writer.Write(3);
```

Comment:
2 should be

Reply:
Wouldn't impact anything, but I'll change it to 2.